src/agents/bee/runners/default/runner.ts

```diff
@@ -295,6 +295,7 @@ export class DefaultRunner extends BaseRunner {
             text: this.templates.system.render({
               tools: await self.system.variables.tools(),
               instructions: undefined,
+              createdAt: undefined,
```

Comment:
just pass a `new Date()` here

Reply:
This results in:
> Type 'Date' is not assignable to type 'string'

So we'll need to track this down.

Reply:
Sorry, I meant `new Date().toISOString()`

Reply:
Done.